tests/console/install_single_package.pm

```diff
@@ -24,6 +24,9 @@ sub run {
     # add specific repository which contains the package
     if (get_var('PACKAGE_REPO')) {
         zypper_call("ar -f " . get_var('PACKAGE_REPO') . " testrepo");
+        # flow suggested in this test requires 'openqa-ci-tools' package
+        # which currently exists in single repo in the world so point to give flexibility here
+        zypper_call("ar -f http://download.suse.de/ibs/Devel:/HPC:/CI/SLE_12_SP2_HPC_CI/ hpc_ci");
```

Comment:
that defeats the purpose of "install_single_package" and even hard-codes a SLE version. Bad approach. And as we managed to define that test module that generic just by use of a test variable you can not even know how it's used by now ;)

Reply:
it does not hardcode SLE version, it is just repo name . This repo will be used for any version of SLE. Let's skip discussion "why repository have such name"  it is not relevant for this PR. 
Once again 'openqa-ci-tools'  exists only in this repository and 'install_single_package' will not work without 'openqa-ci-tools' so I don't see the point to add yet another variable to our already big zoo :).

Reply:
I would rather agree with anton here,
I see no point to add yet another variable for this

Reply:
fine. Can you please rename the test module so that it does not mislead?

Reply:
Suggest the name , please. 
Still can get your point why you think that adding this repo remove any flexibility. This repository required by test backend it does not change anything from 'install_single_package' test perspective. You will still use same variables and generally test flow will be the same. Please describe scenario which WAS available before and not possible right now because adding this repository.